packages/lesswrong/components/messaging/NewConversationDialog.tsx

```diff
@@ -13,7 +17,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   root: {
     maxWidth: 600,
     width: 'min(600px, 100%)',
-    maxHeight: 600,
+    maxHeight: 800,
```

Comment:
I was noticing I wanted to see more results when searching, also in the case where you selected _a lot_ of participants this will give it more room before the space is filled up